plugins/node/opentelemetry-instrumentation-pg/test/pg.test.ts

```diff
@@ -563,5 +563,113 @@ describe('pg', () => {
         client.query('SELECT NOW()').then(queryHandler);
       });
     });
+
+    it('should call postQueryHook with query text if set', async () => {
+      instrumentation.disable();
+      let called = false;
+      const query = 'SELECT NOW()';
+      const config: PgInstrumentationConfig = {
+        postQueryHook: ctx => {
+          called = true;
+          assert.strictEqual(ctx.query, query);
+          assert.strictEqual(ctx.params, undefined);
```

Comment:
These assertions are performed in the hook body.
I think that if they fail, the exception will just be silently ignored and test will pass.
That is because the hook is protected with safeExecuteInTheMiddle which does not propagate exceptions to the caller.